Makefile

```diff
@@ -51,7 +55,7 @@ install-deps:
 	go get -u github.com/alecthomas/gometalinter
 	gometalinter --install
 
-travis: lint test build coverage
+travis: lint test build integration-tests coverage
```

Comment:
I think at this point we should remove this step. We should call each of these separately in the travis.yml file.